example-app/SantokuApp/src/features/qa-event/services/useEvents.ts

```diff
@@ -0,0 +1,9 @@
+import {useGetEvents} from 'features/backend/apis/event/event';
+import {GetEventsParams} from 'features/backend/apis/model';
+import {useState} from 'react';
+
+export const useEvents = (initialParams: GetEventsParams) => {
+  const [listParams, setListParams] = useState(initialParams);
+  const query = useGetEvents(listParams);
+  return {...query, data: query.data?.data, listParams, setListParams};
```

Comment:
The contents of query.data are overwritten and the contents of AxiosResponse cannot be taken. How about giving it another name?

```suggestion
  return {...query, events: query.data?.data, listParams, setListParams};
```

Reply:
Fixed singular
8cad0cc8ed8a2e8b4ec0338e98ef6678ae85d1a8

Reply:
Fixed.
1a3825263daa1b0fc633a500a5d10417f9882b2c